pkgs/applications/misc/kile-wl/default.nix

```diff
@@ -1,20 +1,27 @@
-{ lib, fetchFromGitLab, rustPlatform }:
+{ lib, fetchFromGitLab, rustPlatform, scdoc }:
 
 rustPlatform.buildRustPackage rec {
   pname = "kile-wl";
-  version = "unstable-2021-04-22";
+  version = "unstable-2021-06-01";
```

Comment:
Please add something like https://github.com/NixOS/nixpkgs/blob/f59fb4b41ecd01ed4868922056ddfad6de4ccd94/pkgs/applications/audio/littlegptracker/default.nix#L47 to enable auto updates.

Reply:
Added in b75441c.